src/strategies/jade-smrt/index.ts

```diff
@@ -0,0 +1,131 @@
+import { subgraphRequest } from '../../utils';
+import fetch from 'cross-fetch';
+import { Multicaller, call, getProvider } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'drgorillamd';
+export const version = '1.0.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)',
+  'function totalSupply() external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const block = await provider.getBlock(blockTag);
+  const timestamp = block.timestamp;
+  const avaxBlockTag = await getAvaxBlockTag(timestamp, options);
+
+  // BSC balances:
+  const multiBsc = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address: string) => {
+    multiBsc.call(address+"-jade", options.JADE.address, 'balanceOf', [address]);
+    multiBsc.call(address+"-sjade", options.SJADE.address, 'balanceOf', [address]);
+  });
+
+  // Avax balances:
+  const multiAvax = new Multicaller('43114', getProvider('43114'), abi, { blockTag: avaxBlockTag });
+  addresses.forEach((address: string) => {
+    multiAvax.call(address+"-smrt", options.SMRT.address, 'balanceOf', [address]);
+    multiAvax.call(address+"-smrtR", options.SMRTR.address, 'balanceOf', [address]);
+    multiAvax.call(address+"-smrtRLp", options.SMRTRLP.address, 'balanceOf', [address])
+  });
+
+  // Avax SMRTR/WAVAX pool: SMRTR balance and LP token total supply
+  const LPBalance = await call(getProvider('43114'), abi, [options.SMRTR.address, 'balanceOf', [options.SMRTRLP.address]]);
+  const LPSupply =  await call(getProvider('43114'), abi, [options.SMRTRLP.address, 'totalSupply', []]);
```

Comment:
why not using Multicaller here also like above? 🤔 than two separate calls?

Reply:
Indeed, mb, totally missed the 5 call max -> fixed it (same for multiple coingecko calls), thank you for the feedback:)